### PR TITLE
[QACI-716 QACI-715] - PRTestPipeline - Community 5

### PR DESCRIPTION
**Pull request by cubastanley**
## Description
Creates a barebones pipeline with config for execution using modular pipeline components

### Blockers
Requires shift to Payara Test Library in Jenkins **[PENDING]**

## Documentation
_TBC_

### CODE REVIEW

**Comment by cubastanley** [comment]
Jenkins test

**Comment by cubastanley** [comment]
Modules within code base are successful and working - now pending external suite modules before merge

File: .jenkins/modules/Test/PayaraSamplesTest.groovy [hunk 1]
```diff
@@ -0,0 +1,7 @@
+withMaven(jdk: CFG.jdk, options: [artifactsPublisher(disabled: true)]) {
+    sh """mvn -V -B -ff clean install --strict-checksums -Ppayara-server-remote \
+        -Dpayara.version=${CFG.'build.version'} \
```
Comment (Pandrex247) [inline .jenkins/modules/Test/PayaraSamplesTest.groovy:3]: `-Dpayara.home` needs to be set.

**Review by Pandrex247 — CHANGES_REQUESTED** [review]

**Comment by cubastanley** [comment]
Jenkins test please

**Review by Pandrex247 — APPROVED** [review]

**Pull request merged**